src/main/scripts/terms.validate.js

```diff
@@ -36,6 +36,22 @@ module.exports = async (registry, name) => {
     }
   }
 
+   /* are related terms defined in registry */
+
+  const definedTerms = []
+  for (let t in registry) {
+    let term = registry[t].term
+    definedTerms.push(term)
+  }
+  for (let r in registry) {
+    let relatedTerm = registry[r].relatedTerms
+    for (let rT in relatedTerm) {
+      if (!definedTerms.includes(relatedTerm[rT])) {
+        console.log(name + " registry term '" + registry[r].term + "' contains relatedTerm '" + relatedTerm[rT] + "' that is not a defined term");
```

Comment:
changing this to a `throw` causes build to fail

Reply:
was trying this:
`throw `${name} registry term '${registry[r].term}' contains relatedTerm '${relatedTerm[r]}' that is not a defined term`;`

Reply:
@palemieux disregard. Got it!